src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -90,7 +90,7 @@ const useCoreChatSystemHook = () => {
     const isMyMessage =
       message?.user?.id === signedProfileId || message?.user?.id === anonProfileId;
 
-    const hasUnreadMessage = selectedChannel?.unread_count <= 0;
+    const hasUnreadMessage = selectedChannel?.unread_count >= 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the comparison operator from `<= 0` to `>= 0` for `hasUnreadMessage` seems incorrect. The variable `hasUnreadMessage` should be true when there are unread messages, i.e., when `unread_count > 0`. With the current change, `hasUnreadMessage` will always be true as `unread_count` is always greater than or equal to 0 (assuming it can't be negative). This might lead to incorrect behavior.
```diff
-    const hasUnreadMessage = selectedChannel?.unread_count >= 0;
+    const hasUnreadMessage = selectedChannel?.unread_count > 0;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->